### DISCUSSION TITLE
編集フォームに不自然な空白が表示される

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#510] [要望・アイデア・不具合]
記事の編集フォーム内でずっと下スクロールしていくと、必要以上にスクロールしてしまい、添付画像のような巨大な空白が表示されます。

<img width="1325" alt="Screenshot 2023-08-01 at 18 06 30" src="https://github.com/increments/qiita-discussions/assets/1148320/b2519bac-1649-4532-91a5-0afc162f9b83">

何か致命的な問題があるわけではありませんが、UXはあまり良くないです。。

**Comment by tanaka-asami415** [comment]
@JunichiIto 
ご意見ありがとうございます！

テキストエリア下部に意図的にpaddingを入れております。
理由はスクロール同期を有効にしている場合、プレビューエリア側で最下部までスクロールできなくなってしまうためです。

paddingを入れない状態も試したりもしたのですが、最終的に現在の状態に落ち着きました🙏
ご不便をおかけして申し訳ないですが、他にいい改善方法が見つかるまでは現状のままとなります…🙇

**Reply by JunichiIto** [reply]
ご説明どうもありがとうございます。
うーん、悩ましいですね。。
技術上致し方ない、という開発サイドの理由も理解できるのですが、記事を書いてるときにこういう空白が見えると、一瞬「あれ、こんなに無駄に改行入れまくってたっけ？？」とビックリしちゃうんですよね。

実際、今のUIだと「改行が無駄に入ってるのかどうか」が視覚的にほぼ区別できません。

<img width="1635" alt="blank" src="https://github.com/increments/qiita-discussions/assets/1148320/4803a0f2-67f9-496f-82b4-eccee8f14e48">

「大丈夫だよね？無駄な空行入ってないよね？」という心配と、念のための確認作業が記事を編集するたびに発生するのは、記事を書く人間として体験が良くないので、早めに改善していただきたいなーと思う次第です。

**Reply by tanaka-asami415** [reply]
@JunichiIto 
こちらこそありがとうございます！🙏

確かに改行と空白の区別がつきづらい状態となっていて、執筆時にビックリさせてしまうのは良くないと思います🙇
paddingを入れる事は変わらないと思いますが、空白と改行の違いが区別できるように見た目の改善を検討します！

**Reply by JunichiIto** [reply]
はい、ご検討よろしくお願いします！

**Reply by JunichiIto** [reply]
参考情報ですが、テキストエディタによってはEOF = End of fileを表示するものがあります。

![eoftest](https://github.com/increments/qiita-discussions/assets/1148320/0061fd9d-a9b4-498c-8ef4-ab775ffc83dc)
https://userweb.alles.or.jp/chunichidenko/cpm_mycom177.html

こんな感じの表示を追加するのも一手かもしれません。いちおうご参考までに。

**Reply by shimanakahibiki** [reply]
@JunichiIto
おまたせしました！
空白と改行の違いが区別できるようにする解決策として、
本日エディタで行番号を表示するようにしました！
https://qiita.com/release-notes/2023-10-24

**Reply by JunichiIto** [reply]
対応ありがとうございます！違いがわかりやすくなりました 👍
